themes/origins/sections/slideshow.liquid

```diff
@@ -0,0 +1,279 @@
+{{ 'component-slider.css' | asset_url | stylesheet_tag }}
+{{ 'component-slider.js' | asset_url | script_tag_deferred }}
+
+{% assign slideshow = section.settings %}
+
+<yc-section
+  {% if slideshow.type == 'full' %}
+    data-full
+  {% endif %}
+  style="
+    --section-space-top: {{ slideshow.space_top }}px;
+    --section-space-bottom: {{ slideshow.space_bottom }}px;
+  "
+>
+  <yc-slider
+    {% if slideshow.autoplay %}
+      autoplay
+    {% endif %}
+    {% if slideshow.show_arrows %}
+      arrows
+    {% endif %}
+    {% if slideshow.type != 'full' %}
+      data-radius="true"
+    {% endif %}
+    interval="{{ slideshow.duration | times: 1000 }}"
+    data-type="full"
+    style="
+      --slider-gap: 0;
+      --slider-speed: 600;
+      --slider-per-page: 1;
+      --slider-height: {{ slideshow.height }};
+
+      --section-content-color: {{ slideshow.content_color.hex }};
+      --section-button-background: {{ slideshow.button_background.hex }};
+      --section-button-foreground: {{ slideshow.button_foreground.hex }};
+      --section-overlay-background: {{ slideshow.overlay_background.red }}, {{ slideshow.overlay_background.green }}, {{ slideshow.overlay_background.blue }};
+      --section-arrows-foreground: {{ slideshow.arrows_foreground.hex }};
+      --section-arrows-background: {{ slideshow.arrows_background.red }}, {{ slideshow.arrows_background.green }}, {{ slideshow.arrows_background.blue }};
+    "
+    aria-role="region"
+    aria-roledescription="Carousel"
+  >
+    <div class="slider-area">
+      <div class="slider-box" data-slider-box>
+        <div class="slider-inner" data-slider>
+          {% for block in section.blocks %}
+            {% render 'slideshow-slide' %}
+          {% endfor %}
+        </div>
+      </div>
+    </div>
+
+    {% if section.blocks.size > 1 %}
+      <yc-slider-arrows data-position="{{ slideshow.arrow_position }}">
+        <div class="slider-arrows">
+          <button class="icon" data-arrow="left" aria-label="previous">
+            {% render 'yc-icon', name: 'arrow-left', size: 15 %}
+          </button>
+          <button class="icon" data-arrow="right" aria-label="next">
+            {% render 'yc-icon', name: 'arrow-right', size: 15 %}
+          </button>
+        </div>
+      </yc-slider-arrows>
+    {% endif %}
+  </yc-slider>
+</yc-section>
+
+{%- schema -%}
+{
+  "label": "Slideshow",
+  "templates": ["index", "page"],
+  "settings": [
+    {
+      "type": "select",
+      "id": "height",
+      "label": "Height",
+      "options": [
+        {
+          "value": "420",
+          "label": "Small"
+        },
+        {
+          "value": "560",
+          "label": "Medium"
+        },
+        {
+          "value": "720",
+          "label": "Large"
+        }
+      ],
+      "default": "560"
+    },
```

Comment:
I think using `vh` for the slide height instead of `px` makes it more flexible across screens

Reply:
this is bound by how we calc height in the slider